tf/src/tf_echo.cpp

```diff
@@ -91,14 +91,25 @@ int main(int argc, char ** argv)
   }
   ros::Rate rate(rate_hz);
 
+  int precision(3);
+  {
+    ros::NodeHandle p_nh("~");
```

Comment:
Since we're now potentially using the private nodehandle multiple times. Please just create it at the top and then we can reuse it in all the conditionals.

Reply:
Ok!